docs/blog/pydantic-v2-alpha.md

```diff
@@ -0,0 +1,182 @@
+  :octicons-clock-24: 8 min read
+</div>
+</aside>
+
+---
+
+We're excited to announce the first alpha release of Pydantic V2!
+
+This first Pydantic V2 alpha is no April Fool's joke &mdash; for a start we missed our April 1st target date :cry:.
+After a year's work, we invite you to explore the improvements we've made and give us your feedback. 
+We look forward to hearing your thoughts and working together to improve the library.
+
+For many of you, Pydantic is already a key part of your Python toolkit and needs no introduction &mdash; 
+we hope you'll find the improvements and additions in Pydantic V2 useful.
+
+If you're new to Pydantic: Pydantic is an open-source Python library that provides powerful data parsing and validation &mdash; 
+including type coercion and useful error messages when typing issues arise &mdash; and settings management capabilities.
+See [the docs](/) for examples of Pydantic at work.
+
+## Getting started with the Pydantic V2 alpha
+
+Your feedback will be a critical part of ensuring that we have made the right tradeoffs with the API changes in V2.
+
+To get started with the Pydantic V2 alpha, install it from PyPI. 
+We recommend using a virtual environment to isolate your testing environment:
+
+```bash
+pip install --pre -U "pydantic>=2.0"
+```
+
+Note that there are still some rough edges and incomplete features, and while trying out the Pydantic V2 alpha releases you may experience errors. 
+We encourage you to try out the alpha releases in a test environment and not in production. 
+Some features are still in development, and we will continue to make changes to the API.
+
+If you do encounter any issues, please [create an issue in GitHub](https://github.com/pydantic/pydantic/issues) using the `bug V2` label. 
+This will help us to actively monitor and track errors, and to continue to improve the library’s performance.
+
+This will be the first of several upcoming alpha releases. As you evaluate our changes and enhancements, 
+we encourage you to share your feedback with us.
+
+Please let us know:
+
+* If you don't like the changes, so we can make sure Pydantic remains a library you enjoy using.
+* If this breaks your usage of Pydantic so we can fix it, or at least describe a migration path.
+
+Thank you for your support, and we look forward to your feedback.
+
+---
+
+## Headlines
+
+Here are some of the most interesting new features in the current Pydantic V2 alpha release. 
+For background on plans behind these features, see the earlier [Pydantic V2 Plan](/blog/pydantic-V2/) blog post.
+
+The biggest change to Pydantic V2 is [`pydantic-core`](https://github.com/pydantic/pydantic-core) &mdash;
+all validation logic has been rewritten in Rust and moved to a separate package, `pydantic-core`.
+This has a number of big advantages:
+
+* **Performance** - Pydantic V2 is 5-50x faster than Pydantic V1.
+* **Safety** - Pydantic V2 should have far fewer bugs long term.
```

Comment:
Copy edit suggestion (feel free to ignore if you don't like it): 

> We've made changes to the architecture that we think will help us maintain Pydantic V2 with far fewer bugs in the long term.

Or similar?

Reply:
And I just realized this bullet is basically the same as the next one. Feels like it could use a bit of a rework perhaps?